### DISCUSSION TITLE
Local laptop camera with Docker

### DISCUSSION THREAD

**Discussion by crisperdue** [doitandbedone/ispyagentdvr-docker#601] [Q&A]
I have successfully set up the latest AgentDVR Docker image, run it,
and requested a "Local Device" camera named "Laptop".

This results in lots of errors in the logs:
Error: SourceError: Laptop: CoreLogic.Sources.Combined.MediaStream:Invalid Source ()

If I try to configure a specific local source, nothing is offered.

Is there a simple way to use the local Mac laptop camera with the Docker image,
or is it intended just for use with networked cameras?

Thanks!

**Comment by ispysoftware** [comment]
the docker needs to have access to local devices - you might have more luck installing it directly in the OS instead of in the docker container
